plugin_interpreter/src/rethink_interface.py

```diff
@@ -100,31 +100,6 @@ def start(self, signal):
         self.job_fetcher.start()
         return True
 
-    @staticmethod
-    def connect_to_db(host, port):
```

Comment:
suggest deprecating rather than outright removing for posterity.  

connect_to_db could remain a staticmethod returning connect(host, port)

pragma nocover it because cc is rude